packages/stark-core/src/validation/validators/is-isin/is-isin.validator.fn.ts

```diff
@@ -3,9 +3,14 @@
  */
 const _floor: Function = require("lodash/floor");
 
+/**
+ * @ignore
+ * Name of the validator, in case injection is needed.
+ */
 export const starkIsISINValidatorName: string = "starkIsISIN";
 
 /**
+ * @ignore
```

Comment:
Maybe we can move or copy this block to the place where this function is documented, cause this info seems to be important to undertand how this validation works